src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -57,11 +60,20 @@ public static UserStore getTestInstance(PersistentStorageAgent persistentStorage
 
   /** The in-memory list of Users. */
   private List<User> users;
+  /** The in-memory list of Admins. */
+  private List<User> admins;
 
   /** This class is a singleton, so its constructor is private. Call getInstance() instead. */
   private UserStore(PersistentStorageAgent persistentStorageAgent) {
     this.persistentStorageAgent = persistentStorageAgent;
     users = new ArrayList<>();
+    admins = new ArrayList<>();
+
+    //hard-coded initial Admin:
+    String hashedPassword = BCrypt.hashpw("AdminPass01", BCrypt.gensalt());
+    User iniAdmin = new User(UUID.randomUUID(), "Admin01", hashedPassword, Instant.now());
+    iniAdmin.setAdmin(true);
+    admins.add(iniAdmin);
```

Comment:
This code is doing the same thing as `RegisterServlet.doPost` (well, it will be when you change to using `users` instead of `admins`), so it would be good if it were only in one place (again, avoiding redundancy => fewer bugs).  Since the one on `RegisterServlet` is doing the logic, but then calling this class, let's move the code to a new method on this class and then call the new method from both here and `RegisterServlet.doPost`.  

I suggest implementing it by adding a overload of `addUser` that instead of accepting a `User` object, it takes two strings: `public void addUser(String username, String password)`.

Reply:
Thank you, will be fixed in separate branch!